src/main/java/ch/njol/skript/events/EvtCommand.java

```diff
@@ -30,20 +31,18 @@
 import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
 @SuppressWarnings("unchecked")
-public class EvtCommand extends SkriptEvent { // TODO condition to check whether a given command exists, & a conditon to check whether it's a custom skript command
+public class EvtCommand extends SkriptEvent {
 	static {
-		Skript.registerEvent("Command", EvtCommand.class, CollectionUtils.array(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class), "command [%-string%]")
+		Skript.registerEvent("Command", EvtCommand.class, CollectionUtils.array(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class), "[1:existing] command [%-string%]")
```

Comment:
Might be outside of this scope since it's already a little big but would be nice to have support for multiple commands `[%-strings%]`

Reply:
was something I was thinking about decided to skip it as it was basically already 7am for me it should be within my scope
I'll see what I can do about it

Reply:
You can always make that in another PR \:)

Reply:
eh something that's needed, not sure why we never did it when we first released the event